tests/scripts/run_models.sh

```diff
@@ -23,4 +23,5 @@ export PYTHONPATH=$TT_METAL_HOME
 if [[ $ARCH_NAME == "wormhole" ]]; then
```

Comment:
This branch never executes. Look at the comment above

Please delete these 3 lines. In fact, this script never executes.

Which pipeline would you like to run this in? Post commit? How long is it?

Reply:
@tt-rkim I am trying to include MobilenetV2 in nightly test since MV2 is in `models/experimental` folder. I noticed this script is outdated and removed the changes.